index.js

```diff
@@ -777,4 +777,80 @@ HTTPCache.prototype.repair = function(cb) {
   });
 };
 
+HTTPCache.prototype.clean = function (shouldClean, cb) {
+  var _this = this;
+
+  var numFiles = 0;
+  var fileProgress = 0;
+  var errors = [];
+
+  function checkFile(metaPath, nextCb) {
+    var curFile = fileProgress;
+    fileProgress++;
+
+    if (!/\.meta$/.test(metaPath)) {
+      errors.push('invalid meta path: ' + metaPath);
+      return nextCb();
+    }
+    // Strip off .meta
+    var path = metaPath.slice(0, -5)
+    var meta = null;
+    var shouldCleanResult = null;
+
+    async.series([
+      // Read the meta file.
+      function (cb) {
+        fs.readFile(
+          _this._absPath(metaPath),
+          { encoding: 'utf8' },
+          function (err, contents) {
+            if (err) { return cb(err); }
```

Comment:
It's reeealy helpful to precede errors with filenames because I doubt that they won't propagate to the user. I.e., `return cb("Cannot read " + _this.absPath(metaPath) + ": " + err)`